src/app/feedback/feedback-view.component.ts

```diff
@@ -110,16 +110,19 @@ export class FeedbackViewComponent implements OnInit, OnDestroy {
     );
   }
 
-  editTitle(mode) {
-    this.editTitleMode = mode;
+  toggleTitle(mode) {
+    this.toggleTitleMode = mode;
   }
 
   setTitle() {
     this.couchService.put(this.dbName + '/' + this.feedback._id, this.feedback).subscribe(
-      () => {
-        this.editTitleMode = false;
+      (res) => {
+        this.toggleTitleMode = false;
+        this.couchService.put(this.dbName + '/' + res._rev, this.feedback).subscribe(
```

Comment:
At this point we're done changing values on the database, but we need to modify `this.feedback` so it has the updated value for `_rev`.

Confusingly CouchDB returns that as `res.rev` without the underscore.